tools/actions/create-release-proposal.sh

```diff
@@ -34,6 +34,13 @@ TITLE="$(git log -1 --format=%s)"
 # Use a temporary file for the PR body
 TEMP_BODY="$(awk "/## ${RELEASE_DATE}/,/^<a id=/{ if (!/^<a id=/) print }" "doc/changelogs/CHANGELOG_V${RELEASE_LINE}.md")"
 
+# Handle temporary file length
+MAX_BODY_LENGTH=65536
+if [ ${#TEMP_BODY} -gt $MAX_BODY_LENGTH ]; then
+  echo "TEMP_BODY too long"
+  TEMP_BODY="${RELEASE_DATE}"
+fi
```

Comment:
What do we think about keeping the count in awk to exit early?
```suggestion
TEMP_BODY="$(awk -v MAX_BODY_LENGTH="65536" \
    "/^## ${RELEASE_DATE}/,/^<a id=/{ if (/^<a id=/) {exit;} if (output_length += length(\$0) > MAX_BODY_LENGTH) {exit 1;} print }" \
    "doc/changelogs/CHANGELOG_V${RELEASE_LINE}.md" || echo "…")"
```

Reply:
that makes sense, thx. I can update it if you guys are good with it